src/sentry/api/base.py

```diff
@@ -227,6 +227,8 @@ def paginate(
         # map results based on callback
         if on_results:
             results = on_results(cursor_result.results)
+        else:
+            results = cursor_result.results
```

Comment:
lol

Reply:
one of those "how did this ever work?" moments